### DISCUSSION TITLE
Feature Request: Optionally Enable S3 Transfer Acceleration

### DISCUSSION THREAD

**Discussion by mgcdanny** [duckdb/duckdb#8342] [Ideas]
AWS S3 Transfer Acceleration can speed up transferring data to and from S3 buckets that have the feature enabled.
Perhaps this would speed up reading / writing parquet and csv files directly to and from S3?

https://docs.aws.amazon.com/AmazonS3/latest/userguide/transfer-acceleration-getting-started.html

**Comment by Mause** [comment]
Looking at the documentation you linked, it looks like you can enable it already with just the configuration we already support?

**Reply by mgcdanny** [reply]
Which configuration would that be?

**Reply by mgcdanny** [reply]
Maybe this one `s3_endpoint`
